src/client/Client.js

```diff
@@ -38,7 +38,7 @@ class Client extends BaseClient {
     let data = process.env;
     try {
       // Test if worker threads module is present and used
-      data = require('worker_threads').workerData || data;
+      data = require('worker_threads').workerData ?? data;
```

Comment:
worker_threads is always available, you can just do `const data = workerData ?? process.env;`.